appveyor.yml

```diff
@@ -0,0 +1,25 @@
+version: 1.0.{build}-{branch}
+
+environment:
+  matrix:
+    - RUBY_VERSION: 22
+    - RUBY_VERSION: 21
```

Comment:
Nit: We also test 2.0 and 2.3 on Travis. If that's easy to do it'd be nice to be consistent, but it's not *important*.